ts/observability-kit/src/init.ts

```diff
@@ -0,0 +1,28 @@
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { DocumentLoadInstrumentation } from '@opentelemetry/instrumentation-document-load';
+import { SimpleSpanProcessor, WebTracerProvider } from '@opentelemetry/sdk-trace-web';
+import { HillaEndpointExporter, type HillaEndpointExportMethod } from './HillaEndpointExporter.js';
+
+export function init(
+  endpoint: HillaEndpointExportMethod,
+  exporter: typeof HillaEndpointExporter = HillaEndpointExporter,
+): void {
+  const provider = new WebTracerProvider();
+
+  provider.register({
+    contextManager: new ZoneContextManager(),
+  });
+
+  provider.addSpanProcessor(
+    new SimpleSpanProcessor(
+      new exporter({
+        endpoint,
+      }),
+    ),
+  );
+
+  registerInstrumentations({
+    instrumentations: [new DocumentLoadInstrumentation()],
```

Comment:
Does is make sense to add the same instrumentations as for Flow? (User interaction, XMLHttpRequest, long tasks and our own error instrumentation).

Reply:
Did it in this PR 😅